assembl/templates/index_react.jinja2

```diff
@@ -50,5 +50,18 @@
                 <div id="under-test" class="hidden">V2</div>
             {%- endif %}
         {% endblock %}
+
+        {% if inject_bugherd %}
+            <!-- Bugherd injection -->
+            <script type='text/javascript'>
+            (function (d, t) {
+            var bh = d.createElement(t), s = d.getElementsByTagName(t)[0];
+            bh.type = 'text/javascript';
+            bh.src = 'https://www.bugherd.com/sidebarv2.js?apikey=mtcl06zwfwhfvmxqxb1d5q';
```

Comment:
I would put both the URL and the apikey in the rc files. I would put it in comments in base_env.rc (there are a number of commented keys noted to be filled in client rc files).

Reply:
In the rc file, it could be like:
```
bugherd_url = https://www.bugherd.com/sidebarv2.js?apikey=%(bugherd_api)s
bugherd_api = mtcl06zwfwhfvmxqxb1d5q
```


And here, it would be

```
bh.src = bugherd_url.format(config)
```